packages/react/src/components/NumberInput/NumberInput.tsx

```diff
@@ -611,14 +613,18 @@ const NumberInput = React.forwardRef<HTMLInputElement, NumberInputProps>(
     // AILabel always size `mini`
     const candidate = slug ?? decorator;
     const candidateIsAILabel = isComponentElement(candidate, AILabel);
-    const normalizedDecorator = candidateIsAILabel
+    const normalizedDecorator: ReactNode = candidateIsAILabel
```

Comment:
This type is not required elsewhere. Why is it needed here?

```suggestion
    const normalizedDecorator = candidateIsAILabel
```

Reply:
I don't see an answer to the question. Is the type required, or can it be removed?